cidc_api/services/users.py

```diff
@@ -18,14 +18,25 @@
 from models import Users, CIDCRole
 from emails import new_user_registration, confirm_account_approval
 
+users_api = Blueprint("users", __name__, url_prefix="/users")
+
+
+@users_api.route("/self", methods=["GET"])
+@requires_auth("users.self")
+def get_self():
+    jsonish_user = _request_ctx_stack.top.current_user.__dict__
```

Comment:
Hm, seems kind'a hacky =}
What if sql-alchemy or something will start adding some other field? 
Shouldn't there be like a white-list based on the `models.User._fields` or something like that?

Reply:
Oh, yeah - `Permissions.__table__.columns.keys()` would give the list of valid columns. I'll update to use that.